ci-operator/step-registry/openshift/e2e/aws/capi/openshift-e2e-aws-capi-workflow.yaml

```diff
@@ -2,15 +2,15 @@ workflow:
   as: openshift-e2e-aws-capi
   steps:
     pre:
-    - ref: ipi-install-hosted-loki
-    - ref: ipi-conf
-    - ref: ipi-conf-aws
-    - chain: ipi-install
-    - ref: capi-conf-apply-feature-gate
+      - ref: ipi-install-hosted-loki
```

Comment:
This file seems to only have whitespace changes, perhaps we can drop the changes from this file?